src/js/outreach/views/opt-in.scss

```diff
@@ -44,12 +44,6 @@
   display: block;
   padding: 12px 16px;
   width: 100%;
-
-  &.has-error {
-    border-color: $error-color;
-    color: $error-color;
-    margin-bottom: 6px;
-  }
```

Comment:
We're only doing error styling for inputs on the verify code view. The opt in view doesn't show any errors for input fields. So this styling never gets used.